apps/server/readme.md

```diff
@@ -0,0 +1,21 @@
+# Plant-application
+
+## Installation
+
+1. Install the latest LTS version of Node.js from the official website: (https://nodejs.org)
```

Comment:
Q: what is the minimum of your NOde.js version? please mention. I'm using 18.19.0

Reply:
I can upgrade if reqired

Reply:
I can upgrade if required

Reply:
mine v18.18.0

Reply:
approve give chel